extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -360,6 +362,15 @@ class CornerstoneViewportService extends PubSubService implements IViewportServi
       if (camera) {
         viewport.setCamera(camera);
       }
+      const pan = viewport.getPan();
+      if (flip) {
+        viewport.setCamera({ flipHorizontal: true });
+      }
```

Comment:
Why do you assume `flip` means flipHorizontal, maybe rename it to flipHorizontal then in the HP?